rest_api/config_apis/image-config-openshift-io-v1.adoc

```diff
@@ -126,16 +126,16 @@ Type::
 
 | `domainName`
 | `string`
-| domainName specifies a domain name for the registry In case the registry use non-standard (80 or 443) port, the port should be included in the domain name as well.
+| `domainName` specifies a domain name for the registry In case the registry use non-standard (80 or 443) port, the port should be included in the domain name as well.
 
 | `insecure`
 | `boolean`
-| insecure indicates whether the registry is secure (https) or insecure (http) By default (if not specified) the registry is assumed as secure.
+| `insecure` indicates whether the registry is secure (https) or insecure (http) By default (if not specified) the registry is assumed as secure.
 
 |===
 ..spec.registrySources
 Description::
-  registrySources contains configuration that determines how the container runtime should treat individual registries when accessing images for builds+pods. (e.g. whether or not to allow insecure access).  It does not contain configuration for the internal cluster registry.
+  `registrySources` contains configuration that determines how the container runtime should treat individual registries when accessing images for builds+pods. (e.g. whether or not to allow insecure access).  It does not contain configuration for the internal cluster registry.
```

Comment:
Should "builds+pods" be "builds and pods"?

Reply:
Yep, not sure how I missed this twice when you called it out previously. Fixed.